src/IEC60870-5-104-simulator.Infrastructure/IEC60870-5-104-simulator.Infrastructure.csproj

```diff
@@ -17,4 +17,10 @@
     <ProjectReference Include="..\IEC60870-5-104-simulator.Domain\IEC60870-5-104-simulator.Domain.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Reference Include="Microsoft.AspNetCore.Mvc.Abstractions">
+      <HintPath>..\..\..\..\..\..\Program Files\dotnet\shared\Microsoft.AspNetCore.App\8.0.11\Microsoft.AspNetCore.Mvc.Abstractions.dll</HintPath>
```

Comment:
This looks odd. Is this a local reference to your directory?

Reply:
i removed this part